Blockzilla/Tracking Protection/TrackingProtectionViewController.swift

```diff
@@ -326,7 +326,7 @@ class TrackingProtectionViewController: UIViewController {
         
         preferredContentSize = CGSize(
             width: tableView.contentSize.width,
-            height: tableView.contentSize.height + (headerHeight?.layoutConstraints[0].constant ?? .zero)
+            height: tableView.contentSize.height + (headerHeight?.layoutConstraints[0].constant ?? .zero) + (trackingProtectionItem.settingsValue ? .zero : 50)
```

Comment:
We should not have explicit values for constraints, we should put them in variable, for example in UIConstants.layout